thirdeye/thirdeye-frontend/app/mirage/fixtures/applicationAnomalies.js

```diff
@@ -26,7 +27,68 @@ export default [
     current: 1444,
     baseline: 1000,
     feedback: "Yes, but New Trend",
-    metric: "Metric Name 1",
+    metricName: "Metric Name 1",
```

Comment:
This is starting to become pretty big. not needed in this pr, but we should start considering using scenarios/factories

Reply:
I'll add a `TODO` to refactor this in another PR.